### DISCUSSION TITLE
Terraform Example

### DISCUSSION THREAD

**Discussion by elasticdotventures** [grafana/grafana-infinity-datasource#691] [Ideas]
I'm trying to find any documentation related to configuring the infinity grafana-infinity-datasource with terraform

if such a reference already exists I would appreciate a link to it.
if such a reference does not exist, I'm happy to contribute an example once I get it working. 😉

```
resource "grafana_data_source" "infinity" {
    type = "yesoreyeram-infinity-datasource"
    name = "infinity"

    json_data_encoded = jsonencode({
       ... assembling a list of fields for here.
    })
    secure_json_data_encoded = jsonencode({
    "bearerToken" = "..."
  })
}
```

Is there any place in the existing docs where this information would be kept?  
Terraform (and openTofu, the OSS version) are both popular ways of managing grafana.

**Comment by yesoreyeram** [comment]
Hi, look at the provisioning section in the docs.  Only you need to adjust the values in json data and secure json section.
